osmosis-1/osmosis.zone_assets.json

```diff
@@ -3160,6 +3160,14 @@
       "listing_date_time_utc": "2024-04-13T17:50:00Z",
       "osmosis_unlisted": true,
       "_comment": "Staked Astroport $xASTRO"
+    },
+    {
+      "chain_name": "migaloo",
+      "base_denom": "factory/migaloo1d0uma9qzcts4fzt7ml39xp44aut5k6qyjfzz4asalnecppppr3rsl52vvv/rstk",
+      "path": "transfer/channel-642/factory/migaloo1d0uma9qzcts4fzt7ml39xp44aut5k6qyjfzz4asalnecppppr3rsl52vvv/rstk",
+      "osmosis_verified": true,
+      "listing_date_time_utc": "2024-04-15T17:00:00Z",
```

Comment:
Should this be hidden (visible only in preview mode) until the 15th?

Reply:
I don't really care as long as the stream swap team has what they need